lib/apiservers/service/restapi/handlers/vch_log_get.go

```diff
@@ -139,35 +128,27 @@ func getDatastoreHelper(op trace.Operation, d *data.Data, validator *validate.Va
 	return helper, nil
 }
 
-// getAllLogFilePaths returns a list of all log file paths under datastore folder, errors out when no log file found
-func getAllLogFilePaths(op trace.Operation, helper *datastore.Helper) ([]string, error) {
-	res, err := helper.Ls(op, "")
+// getAllLogs downloads all log files in datastore and concatenates the content
+func getAllLogs(op trace.Operation, helper *datastore.Helper) (string, error) {
+	res, err := helper.Ls(op, "", []string{vchlog.LogFilePrefix + "*" + vchlog.LogFileSuffix}...)
```

Comment:
I think this could be written more simply:

`res, err := helper.Ls(op, "", vchlog.LogFilePrefix + "*" + vchlog.LogFileSuffix)`